src/main/java/org/lisoft/lsml/model/item/AmmoWeapon.java

```diff
@@ -45,25 +45,36 @@ public class AmmoWeapon extends Weapon {
     @XStreamAsAttribute
     private final String ammoTypeId;
     @XStreamAsAttribute
-    private final boolean oneShot;
-
+    private final int volleySize;
+    @XStreamAsAttribute
+    private final int ammoPerShot;
+    @XStreamAsAttribute
+    private final boolean oneShot;    
+    @XStreamAsAttribute
+    private final double firingDelay;
+    
```

Comment:
Could you move the `volleyDelay` attribute from the `Weapon` class here as well? It no longer make sense without `volleyDelay` so it makes sense that they be together here and I don't see any uses of it in the base class anymore.

Reply:
Sure. I’ll move the tests too.

Reply:
done. But volleydelay is an attribute in laser weapons... all zero (to be more pedantic, some 0.0 and some 0 but all zero), and in the PGI file.

Reply:
I don't know of any energy weapon in Battletech that would have volleys. But if there would be one that's added we'd have to move both volley delay and size to the base class... but let's just leave them both in AmmoWeapon for now, it makes more sense.